restorers/model/nafnet/nafnet.py

```diff
@@ -27,6 +35,96 @@ def get_config(self) -> dict:
         return config
 
 
+class BlockStack(keras.layers.Layer):
+    """
+    BlockStack Layer
+
+    Simple utility class to generate a sequential list of same layer
+    """
+
+    def __init__(
+        self,
+        block_class: Type[keras.layers.Layer],
+        num_blocks: int,
+        *args,
+        **kwargs,
+    ) -> None:
+        super().__init__(**kwargs)
+        self.block_class = block_class
+        self.num_blocks = num_blocks
+        self.block_list = []
+        self.args = args
+        self.kwargs = kwargs
```

Comment:
Not necessary.

Reply:
No, it is necessary, because if you see args and kwargs are needed to initialize the model_class that is used in the BlockStack. Hence we need to store those arguments as they are required for constructor.

Reply:
A bit more clarification to my earlier point. I am adding these because I need to save these for get_config method. I could instead use the lost of layers in blockstack to get back those config, i.e. doing something like 
{
...
"kwargs": self.block_list[0].get_config(),
...
}
But I think that would be too roundabout way. Let me know if you want me to update this.